tensorboard/webapp/widgets/range_input/range_input_test.ts

```diff
@@ -426,7 +444,10 @@ describe('range input test', () => {
       minInput.value = '2';
       minInput.dispatchEvent(new InputEvent('change'));
 
-      expect(onSingleValueChanged).toHaveBeenCalledWith(2);
+      expect(onSingleValueChanged).toHaveBeenCalledWith({
+        value: 2,
+        source: RangeInputSource.TEXT,
```

Comment:
There should also be tests for onSingleValueChanged with RangeInputSource.SLIDER.

Reply:
> I would think the single slider event should also have a source. It is emitted from directly from the html file. Also update the test to include that case.

I didn't catch the event emitted from the dom, thanks! I'm moving that to a function as I believe the enum is clearer that way.

Reply:
I've gone ahead and added a test.